src/Header/Header.test.tsx

```diff
@@ -7,8 +7,16 @@ describe('Header', () => {
     render(<Header/>)
   })
 
-  it('should render app name in Header', () => {
-    const header = screen.getByRole('heading', {name: 'Weather Or Not'})
-    expect(header).toBeInTheDocument()
+  it('should render logo and search bar in Header', () => {
```

Comment:
Awesome work. Getting a head start on that testing love it.

Reply:
As Ian said in PD today, 'what's a weakness you won't bring _here_?"